bin/tessel-2.js

```diff
@@ -187,4 +188,22 @@ parser.command('wifi')
   })
   .help('Configure the wireless connection');
 
+parser.command('key')
+  .option('method', {
+    position: 1,
+    required: true,
+    choices: ['generate'],
+  })
+  .callback(function(opts) {
+    key(opts)
+    .then(function(){
+      console.log('then called!');
```

Comment:
I got "then called!" and an error code of 1 which signifies error, though it was successful?